packages/block-library/src/image/edit.native.js

```diff
@@ -432,196 +407,80 @@ export class ImageEdit extends React.Component {
 			wide: 'center',
 		};
 
-		const imageContainerHeight =
-			Dimensions.get( 'window' ).width / IMAGE_ASPECT_RATIO;
-
-		const editImageComponent = ( { open, mediaOptions } ) => (
-			<TouchableWithoutFeedback onPress={ open }>
-				<View style={ styles.editContainer }>
-					<View style={ styles.edit }>
-						{ mediaOptions() }
-						<Icon
-							size={ 16 }
-							icon={ SvgIconCustomize }
-							{ ...styles.iconCustomise }
+		const getImageComponent = ( openMediaOptions, getMediaOptions ) => (
+			<>
+				<TouchableWithoutFeedback
+					accessible={ ! isSelected }
+					onPress={ this.onImagePressed }
+					onLongPress={ openMediaOptions }
+					disabled={ ! isSelected }
+				>
+					<View style={ { flex: 1 } }>
```

Comment:
Hmm. could you please move the `{ flex: 1 }` to const and use it here? Just to have the same reference since right now we create a new object in every call of `getImageComponent`?

Reply:
Updated, I moved it to styles =)